### ISSUE TITLE
Test

### ISSUE THREAD

**Issue by HamidFsian** [potpov/alveolar_canal#2]
Hi,

When testing the model `PosPadUNet3D` with checkpoints you made available (thank's). I am having a Dice=0.0 and Loss=1. Here is the yaml file that I am using. 
```python
title: alveolar_canal_finetuning
project_dir: 'results'
seed: 47

experiment:
  name: Segmentation

data_loader:
  dataset: IAN_Maxillo_dataset\dataset_for_public
  training_set: null
  preprocessing: configs/preprocessing.yaml
  augmentations: configs/augmentations.yaml
  background_suppression: 0
  batch_size: 6
  labels:
    BACKGROUND: 0
    INSIDE: 1
  mean: 0.08435
  num_workers: 4
  patch_shape:
  - 80
  - 80
  - 80
  resize_shape:
  - 168
  - 280
  - 360
  sampler_type: grid
  grid_overlap: 0
  std: 0.17885
  volumes_max: 2100
  volumes_min: 0
  weights:
  - 0.000703
  - 0.999

model:
  name: PosPadUNet3D

loss:
  name: Jaccard

lr_scheduler:
  name: Plateau

optimizer:
  learning_rate: 0.1
  name: SGD

trainer:
  reload: False
  checkpoint: checkpoints\last-seg-pretraining.pth #waht you made available
  do_train: False
  do_test: True
  do_inference: True
  epochs: 100

```

Do you have any idea why it is not working as expected ? 

Regards,
HF

**Comment by LucaLumetti** [comment]
Hi @HamidFsian,
in your config you have set `reload: False` thus, even if you have defined the checkpoints, they are not loaded. If you set `reload: True` everything should work as expected.
Let me know if you need more help, i'll be glad to help!

Regards,
Luca

**Comment by LucaLumetti** [comment]
@HamidFsian in addition to my previous response, it looks like you are using a config file from the [this](https://github.com/AImageLab-zip/alveolar_canal) repository instead of the current one.
The code you will find in the repository I've linked above is up-to-date and I recommend you refer to it.
Of course, if you use a config file given in a different repository, you will encounter many problems and it is expected that it will not work.

**Comment by HamidFsian** [comment]
@LucaLumetti  Thank's for your answers. I have cloned the repo you mentionned. I have changed only some paths. But still having theses metrics when testing the PosPadUNet3D. Here you found the yaml file(seg-pretraining.yaml).

```
title: alveolar_canal_pretrain
project_dir: 'results_segmentation'
seed: 47

experiment:
  name: Segmentation

data_loader:
  dataset: C:\Users\hamid\Desktop\python_projects\mandibular_canal\IAN_Maxillo_dataset\dataset_for_public
  training_set: null # null, generated
  preprocessing: configs/preprocessing.yaml
  augmentations: configs/augmentations.yaml
  background_suppression: 0
  batch_size: 2
  labels:
    BACKGROUND: 0
    INSIDE: 1
  mean: 0.08435
  num_workers: 4
  patch_shape:
  - 80
  - 80
  - 80
  resize_shape:
  - 168
  - 280
  - 360
  sampler_type: grid
  grid_overlap: 0
  std: 0.17885
  volumes_max: 2100
  volumes_min: 0
  weights:
  - 0.000703
  - 0.999

model:
  name: PosPadUNet3D

loss:
  name: Jaccard

lr_scheduler:
  name: Plateau

optimizer:
  learning_rate: 0.1
  name: SGD

trainer:
  reload: True
  checkpoint: checkpoints\last-seg-pretraining.pth
  do_train: False
  do_test: True
  do_inference: False
  epochs: 28

```

```
wandb: Run summary:
wandb:     Epoch 29
wandb: Test/Dice 0.0
wandb:  Test/IoU 0.0
wandb: Test/Loss 1.0
```

Do you have any idea why it is not working ? Thanks

**Comment by LucaLumetti** [comment]
@HamidFsian looks like it is an issue related to our weights, i'll delve more into it and give a feedback to you as soon as possible. Please open an issue on the AImagelab-zip repository in order to have more consistency, i'll close this one as it is not strictly related to this repository.
